Fails to compile with --target=asmjs-unknown-emscripten

When compiling with the newest nightly compiler and `cargo build --target=asmjs-unknown-emscripten`, this happens:

```
   Compiling num_cpus v1.1.0 (file:///path/to/num_cpus)
error[E0425]: unresolved name `get_num_cpus`
  --> src/lib.rs:12:5
   |
12 |     get_num_cpus()
   |     ^^^^^^^^^^^^ unresolved name

error[E0425]: unresolved name `get_num_cpus`
  --> src/lib.rs:28:5
   |
28 |     get_num_cpus()
   |     ^^^^^^^^^^^^ unresolved name
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -145,6 +145,10 @@ fn get_num_cpus() -> usize {
         libc::sysconf(libc::_SC_NPROCESSORS_ONLN) as usize
     }
 }
+#[cfg(target_os = "emscripten")]
+fn get_num_cpus() -> usize {
+    1
+}
 
 #[test]
 fn lower_bound() {

